### DISCUSSION TITLE
useHeader hook does not seem 'native'

### DISCUSSION THREAD

**Discussion by LeoRedin** [infinitered/ignite#2710] [Q&A] [answered]
Hey folks!

I've just got a feedback here from the people I work with and they are saying that the `useHeader` hook creates the headers but the animations don't seem native enough. Is there a way I could make some changes there, for me they look fine enough but there might be something I don't know.

Those animations are mainly screen changes, when we use the header to transition to other pages on our app for example,

Thanks in advance!

**Comment by fpena** [comment]
Hello @LeoRedin!

Thanks for creating this discussion. 

Can you please provide more details as to why your team thinks the header is not native enough and also for expected behaviour? 

Thanks again for opening this, and I am looking forward to hearing from you soon.

**Comment by LeoRedin** [comment]
Hello @fpena! Thanks for replying.

What I got from feedback was mainly related to transitions. Anytime we use our bottom navigator or now specifically the header to go back and fourth on screens the animations on the transitions don't feel like a native app. I'll ask for a video and we could probably check more. ]

I haven't investigated a whole lot but maybe the setOptions overrides something?

**Reply by frankcalise** [reply, marked as answer]
Hey @LeoRedin - do you think this is maybe the side effect you're seeing?

https://github.com/infinitered/ignite/pull/2744
